aiida/cmdline/commands/exportfile.py

```diff
@@ -41,20 +41,33 @@ def cli(self, *args):
     help='Export the given groups by pk')
 @click.option('-g', '--group_names', multiple=True, type=str,
```

Comment:
Could you please also replace underscores with dashes in option names?
This is the way it's done in `aiida/cmdline/params/options` and we should adopt it aiida-wide.

Reply:
Thanks!